templates/alertmanagerconfig.go

```diff
@@ -36,26 +36,14 @@ func convertToAPIExtV1JSON(val interface{}) apiextensionsv1.JSON {
 var _false = false
 
 var pagerdutyAlerts = []string{
-	"NooBaaResourceErrorState",
-	"NooBaaNamespaceResourceErrorState",
-	"NooBaaBucketErrorState",
-	"NooBaaNamespaceBucketErrorState",
```

Comment:
why these checks are removed? newly added alerts cover these alerts also?

Reply:
The ones that are irrelevant to our use case (as decided in a chat with Danny and Liran) are removed, and the others were renamed as per the newer rule file.